app/models/profile.rb

```diff
@@ -54,6 +59,10 @@ def destroy_orphaned_business_objective
     business_objective.destroy if business_objective.profiles.empty?
   end
 
+  def destroy_all_test_results
+    test_results.destroy_all
```

Comment:
Could we make a job out of this and submit it on deletion? The problem I see with this is that unfortunately some pods are OOM when massive deletions like this happen (this would trigger the deletion *with callbacks and associations* of 1500 RuleResults + 10 TestResults assuming reports with 150 rules)

Reply:
We've seen this happening when in QE anyone calls https://github.com/RedHatInsights/compliance-backend/blob/master/app/controllers/systems_controller.rb#L22 - removing all RuleResults is a tough task that can make a pod OOM

Reply:
👍 Will turn this into running in a job.